src/Features/VisualBasic/Portable/IntroduceVariable/VisualBasicIntroduceVariableService_IntroduceLocal.vb

```diff
@@ -36,7 +36,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.IntroduceVariable
                         SyntaxFactory.EqualsValue(value:=expression.WithoutTrailingTrivia().WithoutLeadingTrivia()))))
 
             If Not declarationStatement.GetTrailingTrivia().Any(SyntaxKind.EndOfLineTrivia) Then
```

Comment:
This is different behaviour from C#. declarationStatement does not have this check.

Reply:
As in, this probably causes the eol changes.

Reply:
FWIW, the EOL changes are welcome IMO. The whole point of elastic trivia formatting was precisely for these cases.